src/main/resources/static/js/eiInfo.js

```diff
@@ -1,9 +1,18 @@
 jQuery(document).ready(function() {
     // Fetch injected URL from DOM
-    var frontendServiceUrl = $('#frontendServiceUrl').text();
-    var backendServerUrl = $('#backendServerUrl').text();
+    var frontEndServiceUrl = $('#frontendServiceUrl').text();
+    var frontEndVersion = $('#version').text();
+    var frontEndEnterpriseVersion = $('#enterpriseVersion').text();
+    var frontEndEnterpriseVersionName = $('#enterpriseVersionName').text();
```

Comment:
What do you mean with Enterprise version? This seems mixed with the concept of community/enterprise version of softwares. Maybe this should change name to not cause confusion?

Reply:
Done.